.github/workflows/e2e.yml

```diff
@@ -0,0 +1,72 @@
+name: E2E tests
```

Comment:
Do we need both the e2e.yml and ci.yml? I'd like to just use the ci.yml but it doesn't seem to upload a playwright [report](https://github.com/sunker/explore-logs/actions/runs/8813117735)

Reply:
No the ci.yml should be enough.

Have raised a pr in create-plugin that makes a few slight improvements to the ci workflow template. For example it adds a step that uploads the playwright report (if tests fail) to a GCS bucket. It's pretty helpful. 

Before the jobs in ci.yml
```yml
permissions:
  contents: read
  id-token: write
```

At the very end of ci.yml
```yml
      # Repos in the grafana Github org can publish the report to GCS so that they can be browsed without having to download them.
      # This will make the report public on the Internet, so beware to skip this step if the report contains sensitive information.
      - name: Publish report to GCS
        if: ${{ always() && steps.run-tests.outcome == 'failure' }}
        uses: grafana/plugin-actions/publish-report@main
        with:
          grafana-version: $
          path: playwright-report
```